lib/unexpectedMitm.js

```diff
@@ -730,6 +730,46 @@ module.exports = {
                     return recordedExchanges;
                 });
             })
+            .addAssertion('<any> with http mocked out [and verified] [returning the file] <string> <assertion>', function (expect, subject, filePathWithoutExt) {
+                expect.errorMode = 'nested';
+                var testFile = filePathWithoutExt + '.js';
+
+                if (process.env.UNEXPECTED_MITM_WRITE === 'true') {
+                    return executeMitm(expect, subject).then(function (recordedExchanges) {
+                        var output = 'module.exports = ' +  stringify(recordedExchanges) + ';\n';
+
+                        fs.writeFileSync(testFile, output);
+
+                        if (expect.flags['returning the file']) {
+                            return [recordedExchanges, testFile];
+                        } else {
+                            return recordedExchanges;
+                        }
+                    });
+                }
+
+                return expect.promise(function () {
+                    return require(testFile);
```

Comment:
Hmm, this means that relative paths will be resolved from `node_modules/unexpected-mitm/lib/`, which probably isn't ideal. The most helpful thing would be to do it from the test suite, I don't think that's super easy to accomplish.